### ISSUE TITLE
Will there ever be a proper release?

### ISSUE THREAD

**Issue by ThiloteE** [bugbakery/transcribee#447]
The official funding period has expired and by looking at github commits I see that development has slowed down.
It just pains me to see.

The Audapolis GUI looks nice, but the models it supports are just super bad and outdated.
I expect transcribee to do better, since it at least claims to support whisper (which in itself has also become quite old. If anything, some of its finetunes should be looked at), but there has been no release.

I feel like the momentum is gone and now we are left with two tools that do nothing right instead of one that is properly supported.

**Comment by phlmn** [comment]
Yes :) While we did not make so much progress for some time, we are still committed to the project and gained some traction again. Also, we recently got funding to work on a desktop version of transcribee (still in this repository, stay tuned!). Recent versions of whisper still seems to produce the best results. For the desktop version of transcribee we will also be looking into fine-tuning and other techniques to improve the automated transcription which then will also be added to the web version.

**Comment by phlmn** [comment]
PS. We also recently created proper docker containers to make self-hosted deployments easier. Have a look at `packaging/compose.yml` for an example

**Comment by ThiloteE** [comment]
Thanks for the response. Has been a while. To be honest, I might have been a little too harsh with my critique. I was just frustrated back then. I am sure the two apps did something right back then too and who am I to ask for free things. Anyway, I wasn't quite able to get them to work properly. For transcribee in particular, having to go through lengthy instructions simply to install and start the app was very demotivating. Hope there will be some linux packages (Flatpak, rpm or deb) eventually. Thanks for the pointer to docker. I will try that approach.

Workflow wise, back then my simple idea was to record my thoughts in an audio file and then be able to transcribe that audio file to text. I am easily distracted when sitting on my computer, so thinking about a topic in a calm manner is of big help to me. Another of my needs would have been to feed it scientific articles and then listen to them when I am on my bicycle or when I am having a walk.

Am glad to see the project picking up pace again.
